scripts/package.json

```diff
@@ -60,7 +60,7 @@
     "just-scripts": "^0.38.0",
     "prettier": "~1.17.0",
     "typescript": "3.8.3",
-    "webpack": "^4.29.5",
+    "webpack": "4.42.0",
```

Comment:
Can you make this ^4.42.0 instead? Generally we should try to avoid specific version pinning when possible, let me know if it doesn't work :).